pkgs/development/libraries/qt-6/modules/qtdeclarative/default.nix

```diff
@@ -37,6 +38,12 @@ qtModule {
     })
     # add version specific QML import path
     ./use-versioned-import-path.patch
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
```

Comment:
We should make this none linux conditional and eat the rebuilds in my opinion to notice when it no longer applies as I think all maintainers are not that deep into darwin.

Also nixfmt is unhappy.

Reply:
Fixed the formatting. @K900, are you fine with eating the rebuilds? I can drop the conditional if so.